topics/using-mssa.rst

```diff
@@ -123,7 +127,15 @@ Read the coefficients and the MSSA files
         config = {coefs.getName(): (coefs, keylst[m], [])}
     
         window = int(len(coefs.Times())/2)
-    
+    # Make some parameter flags as YAML.  The defaults should work fine
+    # for most people. 
+    # totPow = True, detrend according to the total power
+    # noMean = True, do not subtract the mean when reading in channels
+    # (noMean = True is valid only for totPow detrending method)
+    # RedSym = True, use the randomized symmetric eigenvalue solver
+    # (RedSym) rather than RedSVD. This toggle is mainly for checking
+    # the accuracy of the default randomized matrix methods 
+
```

Comment:
Hm.  Nothing to do with this commit, but my wording here could be confusing (and I should change the docstrings as well).  Each of these parameters does what the descriptions says *when* they are set to true.  The default for all of them is `false`.   What do you think the best way is to clarify this?

Reply:
Perhaps just listing the default and indicating what happens when it is set to `true` is enough?

I'm debating deleting or heavily editing this page - it is essentially all code, and the mSSA pyEXP tutorials cover the same information. If I edit, I would try to make it as narrative as possible.  Alternatively, if you don't mind mixing theory and examples, I could try to combine the content here with the topics/ssa.... choices to be made.

Reply:
Sure, go for it.  In many ways, you are a better judge of what makes for a better learning experience here. I'm too close to it...